src/components/dropdownSearch/DropdownSearch.tsx

```diff
@@ -152,8 +187,16 @@ export class DropdownSearch extends React.Component<IDropdownSearchProps, {}> {
         return options.length ? options : this.getNoOptions();
     }
 
+    protected getSelectedOption(): IDropdownOption {
+        return _.findWhere(this.props.options, {selected: true});
+    }
+
+    protected getSelectedOptions(): IDropdownOption[] {
+        return _.where(this.props.options, {selected: true});
+    }
+
     protected getDropdownOptionAppend(option?: IDropdownOption): JSX.Element {
-        return !_.isUndefined(option && option.append)
+        return option && option.append
             ? <div className='dropdown-option-append'>{option && option.append}</div>

```

Comment:
you can write `<div className='dropdown-option-append'>{option.append}</div>` since we already know it's defined at this point